packages/cli/runtime.js

```diff
@@ -0,0 +1,29 @@
+'use strict';
+
+const {
+  render,
+  Header,
+  Import,
+  Miss,
+  Status,
+  ServerDataContext,
+  withServerData,
+  ConfigContext,
+  withConfig,
+} = require('hops-react');
+
+const { Mixin, strategies } = require('hops-mixin');
+
+module.exports = {
+  Header,
+  Import,
+  Miss,
+  Status,
+  render,
+  ServerDataContext,
+  withServerData,
+  ConfigContext,
+  withConfig,
```

Comment:
And here as well?

Reply:
As discussed, we won't include the `hops-config` for now in the default preset.